app/src/main/res/values/strings.xml

```diff
@@ -13,4 +13,5 @@
     <string name="title_pressure">Pressure</string>
     <string name="title_current_detail">Current Detail</string>
     <string name="title_load_failure">Load data failure</string>
+    <string name="title_no_data">No data!</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_no_data` appears to be unused
 Unused resources make applications larger and slow down builds.